### PR TITLE
Configuration: Huge pages support

### PR DESCRIPTION
**Pull request by mcastelino**
Enable huge pages support at a node level. This results in all
containers on that node being launched using huge pages.

This is useful when these nodes are used for NFV workloads
where there will be user mode networking stacks in use on that
host or within the container.

Fixes https://github.com/clearcontainers/runtime/issues/626

Signed-off-by: Manohar Castelino <manohar.r.castelino@intel.com>

### CODE REVIEW

File: config.go [hunk 1]
```diff
@@ -232,7 +233,7 @@ func newQemuHypervisorConfig(h hypervisor) (vc.HypervisorConfig, error) {
 		DefaultVCPUs:          h.defaultVCPUs(),
 		DefaultMemSz:          h.defaultMemSz(),
 		DisableBlockDeviceUse: h.DisableBlockDeviceUse,
-		MemPrealloc:           h.MemPrealloc,
```
Comment (grahamwhaley) [inline config.go]: can you expand on why the MemPrealloc change as well pls?

**Comment by grahamwhaley** [comment]
just to note - CI's and QA failing no doubt due to pending re-vendoring. Maybe we should mark such PRs as WIP until we know the missing parts have landed so we don't invoke CI on things we know will fail etc.

File: config.go (hunk 1 above)
Comment (mcastelino) [reply config.go]: @grahamwhaley no, that is a bug. I will fix it. The two are independent.

**Comment by sameo** [comment]
@mcastelino Could you please rebase that one? It should pass now.

**Comment by devimc** [comment]
lgtm

**Pull request merged**